docs/wxm-devices/D1-bundle/D1-connect-and-claim.mdx

```diff
@@ -0,0 +1,190 @@
+If unsure, check the sticker under the station or the box that came in it.
+:::
+
+
+<hr/>
+
+### Connect D1 Gateway to WiFi
+
+<img 
+  src={useBaseUrl("/img/wxm-devices/D1-bundle/D1-Wifi.jpg")} 
+  style={{ width: '200px', height: 'auto', objectFit: 'contain' }} 
+/>
+
+
+By now, your D1 Gateway is set and ready for claiming!
+Swipe left to navigate to the settings screen. There, tap on "Configure WiFi" to switch to connect mode.
+
+Once this is done, connect to D1 Gateway via WiFi (you should see in the networks screen a "WXM_WG1200").
+
+A popup will open, asking you to select the desired network you'd like to connect the D1 Gateway. Select the one you want and provide the password for it.
+
+In case of a successful connection, and the popup does't open, navigate with your browser to 10.10.0.1 and follow the instructions displayed. 
+
+:::note
+Once the WiFi credentials have been successfully transferred to D1 Gateway, the device will reboot and the WiFi icon will turn green!
+:::
+
+<hr/>
+
+### Enter the D1 Gateway Claiming Process
+<img 
+  src={useBaseUrl("/img/wxm-devices/D1-bundle/D1-Wifi.jpg")} 
+  style={{ width: '200px', height: 'auto', objectFit: 'contain' }} 
+/>
+
+- Select the Claim Device option, a QR code will appear on the screen.
+
+Next, you will need to scan the QR code that appears on the D1 Gateway screen with your WeatherXM app.
+
+<img 
+  src={useBaseUrl("/img/wxm-devices/D1-bundle/D1-QR.jpg")} 
+  style={{ width: '200px', height: 'auto', objectFit: 'contain' }} 
+/>
+
+### Download WeatherXM app and login
+
+
+<div class="explanation">
+  <div class="app-store">
+   <a href="https://play.google.com/store/apps/details?id=com.weatherxm.app&hl=en&gl=US" target="blank"><img src={useBaseUrl("img/wxm-devices/wifi-m5-bundle/google-play-badge.svg")}/></a>
+   <a href="https://apps.apple.com/ie/app/weatherxm/id1629841929" target="blank"><img src={useBaseUrl("img/wxm-devices/wifi-m5-bundle/App_Store_Badge.svg")}/></a>
+
+  </div>
+  <span>Download our WeatherXM apps from their respective app store. Once you have your WeatherXM app, login to your account (you have to create a new WeatherXM account).</span>
+</div>
+
+<hr/>
+
+### WeatherXM App - Claim Device
+
+
+<div class="explanation">
+        <img style={{width:"calc(100%*1/3 - 10px)"}} src={useBaseUrl("/img/wxm-devices/D1-bundle/select-your-station.png")} />
+  <div style={{width:"calc(100%*2/3 - 10px)"}} >
+
+
+#### Follow the steps from the app to successfully claim your D1 Gateway.
+
+
+
+
+  </div>
+</div>
+
+<hr/>
+
+
+### Confirm station installation Location & Claim your Device
+
+<img 
+  src={useBaseUrl("/img/wxm-devices/D1-bundle/D1-Location.png")} 
+  style={{ width: '200px', height: 'auto', objectFit: 'contain' }} 
+/>
+
+
+#### Select the Location that you'll Install your Station
+
+Allow access to your location so that the WeatherXM app knows where you are planning to install your weather station. Doing so will make it easier to complete this step.
+
```

Comment:
This whole "Confirm station installation Location & Claim your Device" structure should be like M5's, which means the image on the left, the text on the right and not one below another like it's now.